python/cuml/cuml/decomposition/pca.pyx

```diff
@@ -173,9 +173,6 @@ class PCA(UniversalBase,
         1 2.333...
         2 2.333...
         dtype: float32
-        >>> print(f'noise variance: {pca_float.noise_variance_}') # doctest: +SKIP
-        noise variance: 0   -7.377287e-08
-        dtype: float32
```

Comment:
Why not update instead of remove?

Reply:
Just didn't seem useful in the example. It's a rarely used attribute and already listed lower down as a fitted attribute.